apps/keyboard/test/marionette/uninstallation_test.js

```diff
@@ -56,10 +56,16 @@ marionette('Show Keyboard App after uninstallation', function() {
   function check3rdPartyIme() {
     // switch back to system
     client.switchToFrame();
-    systemInputMgmt.switchToActiveKeyboardFrame();
+
+
+    // wait for the 2nd keyboard is loaded
+    var inputWindows = systemInputMgmt.inputWindows;
     client.waitFor(function() {
-      return imeTestApp.sendKeyButton.displayed();
+      return inputWindows.length > 1;
     });
+
+    systemInputMgmt.switchToActiveKeyboardFrame();
+    return imeTestApp.sendKeyButton.displayed();
```

Comment:
Should we move the original `client.waitFor()` here or we can sure of when the input window is there the first value of `imeTestApp.sendKeyButton.displayed()` is always truthy?